app/Http/Controllers/EntityCreatorController.php

```diff
@@ -95,9 +97,18 @@ public function store(Request $request, Campaign $campaign, EntityType $entityTy
         // Handle dynamic elements
         $this->inputFields = $values;
         $this->dynamicTags()
+            ->dynamicParent($entityType)
```

Comment:
![image](https://github.com/user-attachments/assets/95dee7b5-16f1-4873-9386-133b19199661)
Doesn't work on orgs, maps, timelines, races. Didn't check the rest